projects/env-semconv.md

```diff
@@ -0,0 +1,66 @@
+### Description
+
+This working group looks to identify and standardize observability for
+sustainability purposes with regards to infrastructre and applications. This
+would look at current tooling around sustainability metrics and standardize the
+meaning and approach for these metrics
+
+
+current efforts of implementations:
+
+- [Kepler](https://github.com/sustainable-computing-io/kepler)
+- [Scaphandre](https://github.com/hubblo-org/scaphandre)
```

Comment:
Would people from these projects be willing to be involved here? Do you know if they would be willing to standardize on OTel semantic conventions?

Reply:
We can talk with them, and find out, they do have a relationship with TAG ENV

Reply:
It the Kepler project is interested, I'll push for someone to maybe join

Reply:
@pyohannes and @Spazzy757 this could be indeed interesting from the Kepler standpoint

However, some time ago, there was a [presentation during the Kepler community](https://github.com/danielm0hr/edge-metrics-measurements/blob/main/talks/DanielMohr_PromAgentVsOtelCol.pdf) that included an overhead comparison between the OpenTelemetry (OTel) and Prometheus SDKs, showing that the OTel SDK had significantly higher overhead.

As you know, the key difference between these SDKs is how they export metrics. With OpenTelemetry, instrumentation libraries within the application actively push data to backend systems, whereas the Prometheus SDK relies on applications to expose HTTP endpoints for scraping by a monitoring server. These approaches represent orthogonal push and pull approaches.

However, while the Prometheus SDK does offer the option to [enable the OpenMetrics format](https://github.com/prometheus/client_golang/blob/main/examples/exemplars/main.go#L64), it still follows a pull-based approach. 

Given that performance is very important to us, my questions are: do we really need to migrate from Prometheus SDK to OTel SDK? Has the overhead of the OpenTelemetry SDK been improved?

Reply:
Hi @marceloamaral, I just wanted to mention that, in those experiments shared in the community, I don't think the OTel SDKs were used. (Please, @danielm0hr, correct me if I'm wrong). Metrics are produced with avalanche in prometheus format and, then, scraped with different mechanisms. (Prometheus, or OpenTelemetry collector in different setups).

My main takeaway from this work is that, when collecting prometheus metrics, (by the time the experiments were made) it was more efficient to just use the prometheus tooling. I don't have proof if this would be the sole contribution for the overhead in usage but my suspicions are pointing to the fact that the OpenTelemetry collector needed to transform from prometheus to OpenTelemetry formats back and forth in this experiment (which is still very valid on its own, but I don't think it will apply for the SDK performance discussion).

For the (very relevant, and important) points that you are raising for assessing the viability of changing the SDK, I think the comparison should be between: 1) metrics generated with the OTel SDK (with the OTel pipeline) vs  2) the current prometheus approach. 

BTW, changing to the OpenTelemetry SDK should not be an impediment to continue using prometheus metrics ( see [prometheus exporter](https://opentelemetry.io/docs/languages/go/exporters/#prometheus-experimental) ).

I hope this helps a bit! We can also follow up in the Kepler community if needed!

Reply:
The proposal of this issue is also not related to the SDKs, but to agree on a common convention for sustainability-related metrics.

Reply:
@jgomezselles, thanks for the clarification! It will indeed be interesting to have another experiment comparing the overhead of the OTel versus Prometheus SDK. Any volunteer? :)

Reply:
@dmathieu, right, that makes sense! 
I'd be happy to participate in the discussion.

Reply:
@marceloamaral I have added you to the list of engineers